addons/mass_mailing_crm/views/mailing_mailing_views.xml

```diff
@@ -22,4 +22,17 @@
     		</xpath>
         </field>
     </record>
+
+    <record id="mailing_mailing_lead_view_search" model="ir.ui.view">
```

Comment:
What is the purpose of this new view, primary, with priority 1 (aka will be used by default) ? Just see a by day instead of a by month ? Is that really relevant ? If we really want a specific view for that, maybe set a higher priority (aka less default) and ensure action in mass mailing uses it (and not other actions).